.circleci/config.yml

```diff
@@ -30,6 +31,15 @@ jobs:
           paths:
             - "/tmp/alexa/*"
 
+      - run:
+          name: Check Similarweb ranking
+          command: ruby ./tests/similarweb.rb
```

Comment:
The CircleCI jobs don't actually run `bundle install` at the moment as all their required gems come preinstalled.
Therefore, this command would need to also include installation of the required gems (mainly 'addressable' and 'parallel' if using my suggested changes).

Reply:
Isn't that what "Gemfile" is for?

Reply:
The Gemfile tells Bundler what to install but it doesn't install things without `bundle install` being executed.